cas-server-webapp/pom.xml

```diff
@@ -79,6 +79,11 @@
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-web</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.pac4j</groupId>
+      <artifactId>pac4j-http</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
I want to make sure this does not result into any conflicts or extra dependencies. Have you tested this too?